.idea/misc.xml

```diff
@@ -42,11 +45,13 @@
           <item index="12" class="java.lang.String" itemvalue="io.reactivex.annotations.NonNull" />
           <item index="13" class="java.lang.String" itemvalue="io.reactivex.rxjava3.annotations.NonNull" />
           <item index="14" class="java.lang.String" itemvalue="org.jspecify.nullness.NonNull" />
+          <item index="15" class="java.lang.String" itemvalue="org.jspecify.annotations.NonNull" />
+          <item index="16" class="java.lang.String" itemvalue="jakarta.annotation.Nonnull" />
         </list>
       </value>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_21" default="true" project-jdk-name="jbr-21" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_17" default="true" project-jdk-name="17" project-jdk-type="JavaSDK">
```

Comment:
Was the downgrade to JDK 17 done intentionally?

Reply:
Yes, because we build the releases with debian 12 bookworm.
Java 21 was an accident when I used MacOS to make the Android Studio Update,

Reply:
Got it!